plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeUtils.java

```diff
@@ -103,6 +106,14 @@ public static Object readNativeValue(Type type, Block block, int position)
             int nanoAdjustment = floorMod(epochMicros, MICROSECONDS_PER_SECOND) * NANOSECONDS_PER_MICROSECOND;
             return DATETIME_FORMATTER.format(toZonedDateTime(epochSeconds, nanoAdjustment, UTC));
         }
+        if (type.equals(TIMESTAMP_TZ_MILLIS)) {
```

Comment:
This branch isn't tested in my understanding.

Reply:
Yeah this one doesnt make sense. removing